TTHAnalysis/python/plotter/susy-ewkino/2lss/plots_crtnl.txt

```diff
@@ -0,0 +1,134 @@
+tot_weight: 1 : 1,0.5,1.5; XTitle = "Selected events", Legend='TR', MoreY=2.0
```

Comment:
There is already a plots_2lss file. Could you please confirm that I should just rename susy-ewkino/2lss/plots_crtnl.txt overwriting the current plots_2lss.txt?